docs/book/src/test_add_more.md

```diff
@@ -68,7 +68,7 @@ Now we update our command logic to return an error when this situation arises:
             BankAccountCommand::WithdrawMoney { amount } => {
                 let balance = self.balance - amount;
                 if balance < 0_f64 {
-                    return Err(AggregateError::new("funds not available"));
+                    return Err(BankAccountError(String::from("funds not available")));
```

Comment:
seems this can be simplified to 
`Err(BankAccountError::from("funds not available"))`

Reply:
Thank you for your priceless feedback. I did add a commit to take It into account.